dulwich/refs.py

```diff
@@ -882,3 +882,9 @@ def write_info_refs(refs, store):
 
 def is_local_branch(x):
     return x.startswith(b'refs/heads/')
+
+
+def strip_peeled_refs(refs):
```

Comment:
Please add a basic test for strip_peeled_refs.

Reply:
Should I create a new test class for this test, or is there an existing one where this would be appropriate?

Reply:
Test added.